### PR TITLE
Shuzhi

### PR DESCRIPTION
**Pull request by non-convex**
Add multi-band blending code

### CODE REVIEW

File: src/options/swap_options.py [hunk 1]
```diff
@@ -11,6 +11,7 @@ def initialize(self):
 		self.parser.add_argument('--num_seg_cls', type=int, default=12,help='Segmentation mask class number')
 		self.parser.add_argument('--train_G', default=True, type=bool, help='Whether to train the model')
 		self.parser.add_argument('--device', default='cuda:0', type=str, help='Which GPU(s) to use')
+		self.parser.add_argument('--lap_bld', action='store_true', help='Whether to use Laplacian multi-band blending')
```
Comment (e4s2022) [inline src/options/swap_options.py:14]: How bout setting this option to True by default?

File: start.sh
Comment (e4s2022) [inline start.sh:1]: delete this file.

**Review by e4s2022 — CHANGES_REQUESTED** [review]
Thanks for the work. Please review the comments and revisit them.

File: src/options/swap_options.py (hunk 1 above)
Comment (non-convex) [reply src/options/swap_options.py:14]: Now the blending result is not good, it is better to inpainting the mask mismatch area (represented by hole_map in the code), and then use the blending

Comment (e4s2022) [reply src/options/swap_options.py:14]: Add this option to README

**Pull request merged**